mayaUSD.mod.template

```diff
@@ -1,32 +1,37 @@
-+ USD ${USD_VERSION} ${PXR_USD_LOCATION}
++ USD ${USD_VERSION} ${USD_INSTALL_LOCATION}
+icons: 
+plug-ins: 
+presets: 
+scripts: 
+resources: 
 PYTHONPATH+:=lib/python
-PATH+:=bin
-PATH+:=lib
```

Comment:
These PATH vars are no needed on Linux/Mac. There is a Windows specific .mod template now.